docs/orchestrate/getting-started.md

```diff
@@ -11,6 +11,8 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 This page outlines how to run your browser and mobile tests in Sauce Orchestrate.
 
 ## What You'll Need
+:::caution Please ensure your organization is enrolled to use Sauce Orchestrate otherwise you will encounter an error. If you are not enrolled, please feel free to reach out to your Customer Success Manager or Support at help@saucelabs.com :::
+
```

Comment:
We could move the note higher up the page.
```suggestion
```

Reply:
wherever is best